docs/fsharp/getting-started-netcore.md

```diff
@@ -0,0 +1,315 @@
+---
+title: Building simple Fsolutions with the .NET CLI
+description: Building simple Fsolutions with the .NET CLI
+keywords: .NET, .NET Core
+author: cartermp
+manager: wpickett
+ms.date: 06/20/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: 615db1ec-6ef3-4de2-bae6-4586affa9771
+---
+
+Getting started with F# on .NET Core
+====================================
+
+This article covers how you can get started with using F# on .NET Core with the .NET Core Preview 2 SDK.  It will go through building a multi-project solution with a Class Library, Console App, and xUnit test project.
+
+Prerequisites
+-------------
+
+To begin, you must install the [.NET Core Preview 2 SDK](https://www.microsoft.com/net/core).
+
+This article uses the Command Line and [Visual Studio Code](https://code.visualstudio.com) as the text editor.  You can use any editor you like.
+
+Building a Simple Multi-project Solution
+----------------------------------------
+
+1. Open up a Command Line/Terminal.
+2. Create a new directory named `FSNetCore`.  Open Visual Studio code or your preferred editor inside this directory.  If you haven't already, 
+3. Under `FSNetCore`, create `src` and `test` directories.
+4. Under `FSNetCore`, create a new file called `global.json`.  It should have this as its contents:
+
+```json
+{
+    "projects":[ "src", "test" ]
+}
+```
+
+Your solutions structure should now look like this:
+
+```
+FSNetCore/
+|---src/
+|---test/
+|---global.json
+```
+
+### Writing a Class library
+
+1. Create a `Library` folder under `FSNetCore/src`.
+2. From `FSNetCore/src/Library`, `dotnet new -l F#`.
```

Comment:
it would be good to add context to these steps. Here you are back to command line no? and you're executing `dotnet new ...`